packages/webapp/webapp_server.js

```diff
@@ -815,7 +816,55 @@ var runWebAppServer = function () {
     var localPort = WebAppInternals.parsePort(process.env.PORT) || 0;
     var host = process.env.BIND_IP;
     var localIp = host || '0.0.0.0';
-    httpServer.listen(localPort, localIp, Meteor.bindEnvironment(function() {
+
+    if (typeof localPort == "number")
+    {
```

Comment:
Bit of a nit pick, but can you follow the existing opening/closing brace style that's used in the rest of the file (open brace on same line, etc.)?